test/Atmos/Parameterizations/Microphysics/KM_ice.jl

```diff
@@ -667,7 +667,7 @@ function main()
 
     # time stepping
     t_ini = FT(0)
-    t_end = FT(5 * 60) #FT(4 * 60 * 60) #TODO
+    t_end = FT(4 * 60 * 60) #TODO
```

Comment:
Rebase artifact?

Reply:
Yes it is, a rebase should resolve this.